src/cascadia/TerminalApp/TerminalPage.xaml

```diff
@@ -132,5 +139,14 @@ the MIT License. See LICENSE in the project root for license information. -->
                      IsOpen="False"
                      Severity="Warning"
                      Message="{x:Bind KeyboardServiceDisabledText, Mode=OneWay}"/>
+
+        <!-- A TeachingTip with IsLightDismissEnabled="True" will immediately
+        dismiss itself if the window is unfocused (In Xaml Islands). This is
+        tracked by MUX#4382-->
+        <mux:TeachingTip x:Name="WindowIdToast"
+                         IsLightDismissEnabled="True"
+                         Title="{x:Bind WindowId, Mode=OneWay, Converter={StaticResource WindowIdConverter}}"
+                         Subtitle="{x:Bind WindowName, Mode=OneWay, Converter={StaticResource WindowNameConverter}}">
```

Comment:
Shouldn't we reverse these? Title = Name, subtitle = number? Because the user explicitly assigned a name, so we should present that information first?

Reply:
Oh see, I was mostly thinking about the (majority) case where windows aren't named:
<hr>

#### \<unnamed-window>
Window: 1

<hr>

vs

<hr>

#### Window: 1
\<unnamed-window>

<hr>